contracts/VaultFactory.vy

```diff
@@ -68,13 +75,13 @@ struct PFConfig:
     fee_recipient: address
 
 # Identifier for this version of the vault.
-API_VERSION: constant(String[28]) = "3.0.1"
+API_VERSION: constant(String[28]) = "3.0.2"
 
 # The max amount the protocol fee can be set to.
 MAX_FEE_BPS: constant(uint16) = 5_000 # 50%
 
 # The address that all newly deployed vaults are based from.
-VAULT_BLUEPRINT: immutable(address)
+VAULT_ORIGINAL: immutable(address)
```

Comment:
did you consider `VAULT_IMPLEMENTATION`?

Reply:
i did not.

I guess that would be more accurate.

Went with language more associated with just cloning than proxies

I guess Implementation may be confusing, and seem like its possible to upgrade it.